packages/composables/src/logic/useIntercept.ts

```diff
@@ -36,6 +36,10 @@ export const INTERCEPTOR_KEYS = {
   USER_LOGOUT: "onUserLogout",
 };
 
+export interface IInterceptorCallbackFunction {
```

Comment:
I'm not sure about this naming at all 😅

Reply:
I like the Java-like interfaces (or even php naming convention ;) )

Reply:
and please mark all exported things as @beta or @alpha... depending on the destination